Dockerfile

```diff
@@ -14,6 +12,12 @@ USER zulip
 
 RUN ln -nsf /srv/zulip ~/zulip
 
-RUN echo 'export LC_ALL="en_US.UTF-8"' >> ~zulip/.bashrc
+RUN echo 'export LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8"' >> ~zulip/.bashrc
 
 WORKDIR /srv/zulip
+
+CMD ["/usr/bin/python /srv/zulip/tools/provision.py","--docker"]
+CMD ["source /srv/zulip-venv/bin/activate"]
+CMD ["./srv/zulip/tools/start-dockers"]
```

Comment:
Hey Thanks for Reviewing! 

`start-dockers` is a bash script so the CMD is to invoke the https://github.com/zulip/zulip/blob/master/tools/start-dockers. 

I attempted to confirm but I have some other misunderstanding with docker that the path and source code doesn't exist when I attempt to pack a new image. 

I'm going to roll back the  CMD changes and keep this to just the env LANG changes and make  two separate contributions.